src/App.css

```diff
@@ -1 +1,23 @@
+.App {
+  text-align: center;
+}
+button {
+  border: 1px solid red;
+  background-color: white;
+  color: black;
+  padding: 10px;
+  margin: 10px;
+  font-size: 20px;
+  cursor: pointer;
+}
 
+button:hover {
+  background-color: red;
+  color: white;
+}
+
+button:active {
+  border: 1px solid white;
+  background-color: red;
+  color: white;
+}
```

Comment:
Let's move this a separate file `SurveyQuestion.css`, import it in `SurveyQuestion.js`, add use class names instead of the `button` selector (same reason as for `img` and `h2` element selectors, they are not specific enough)